### ISSUE TITLE
Ensures <img> elements have alternate text or a role of none or presentation: A11y_debug-adapter-protocol_AI4W

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/debug-adapter-protocol#509]
"**Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix"**

[Check out Accessibility Insights!](https://github.com/microsoft/debug-adapter-protocol/issues/url) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

**Github Tags:** #A11yMAS; #A11ySev2; #A11yTCS; #AI4W; #Benchmark; #ChromiumEdge; #DesktopWeb; #FTP; #WCAG 1.1.1; #image-alt; #Win11;  #P2_debugadapterprotocol_Web_Sep2021; #Visual Studio Code Client; #Closed;

## Environment Details:
Application Name: Debug Adapter Protocol Page
#URL: https://microsoft.github.io/debug-adapter-protocol/specification
Windows 11 Enterprise Version 23H2 OS build 22631.4317
Chromium Edge Version 129.0.2792.89 (Official build) (64-bit)

## Repro Steps:

1. Launch the url: https://microsoft.github.io/debug-adapter-protocol/overview
2. Run accessibility insights for web and see the results.

## Issue:
Ensures <img> elements have alternate text or a role of none or presentation (image-alt - https://accessibilityinsights.io/info-examples/web/image-alt)

Target application: Specification - Overview - https://microsoft.github.io/debug-adapter-protocol/overview

Element path:  img[src$="init-launch.svg"]

Snippet: <img src="./img/init-launch.svg" width="100%"

## How to fix:

Fix any of the following:

Element does not have an alt attribute
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attribute
  Element's default semantics were not overridden with role="none" or role="presentation"

Environment: Microsoft Edge version 129.0.0.0

====

This accessibility issue was found using Accessibility Insights for Web 2.44.0 (axe-core 4.9.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

## WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/non-text-content

## Attachment:
![Image](https://github.com/user-attachments/assets/62316ed4-1bdb-437c-be2e-0aab6e2e438a)

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:12-03-24;

Closing this bug as tracking under new service tree id.
Tracking bug id: https://github.com/microsoft/monaco-editor/issues/4770
